inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -731,14 +732,16 @@ def handle_updates(dispatcher: Any,
             dispatcher._send_result(f"Error parsing schedule manifest: {str(e)}", request_id)
             return
         
-        # TODO: Change single and repeated to add the schedules to the scheduler DB
+        # Add schedules to the database
         if schedule.single_scheduled_requests or schedule.repeated_scheduled_requests:
-            sql_lock.acquire()
-            for single_task in schedule.single_scheduled_requests:
-                SqliteManager().create_task(single_task)
-            for repeated_task in schedule.repeated_scheduled_requests:
-                SqliteManager().create_task(repeated_task)
-            sql_lock.release()
+            def process_scheduled_requests(scheduled_requests: Sequence[Schedule]):
+                with sql_lock:  
+                    sqliteManager = SqliteManager()
+                    for requests in scheduled_requests:
+                        sqliteManager.create_schedule(requests)
+            all_scheduled_requests = schedule.single_scheduled_requests + schedule.repeated_scheduled_requests                
+            process_scheduled_requests(all_scheduled_requests)
+
             dispatcher._send_result("Scheduled requests added.", request_id)
```

Comment:
this would be interpreted as an error by cloudadapter. convention is send "" for acknowledge/success

Reply:
thanks for the reminder.  I'll change it.